apis/apps/v1alpha1/cluster_types.go

```diff
@@ -652,6 +652,16 @@ type ClusterComponentSpec struct {
 	// +optional
 	EnabledLogs []string `json:"enabledLogs,omitempty"`
 
+	// Specifies Annotations to override or add for underlying Pods.
+	//
+	// +optional
+	Annotations map[string]string `json:"annotations,omitempty"`
+
```

Comment:
Is there any specific reason for not supporting user-defined labels here?

Reply:
https://github.com/apecloud/kubeblocks/discussions/7557#discussioncomment-9814839

just saw one.